docs/change_log/index.md

```diff
@@ -3,6 +3,11 @@ title: Change Log
 Python-Markdown Change Log
 =========================
 
+_______, 2019: Released version 3.1.1 (a bug-fix release).
+
+* Fixed import failure in setup.py when the source directory is not
```

Comment:
Maybe wrap `setup.py` and `sys.path` in backticks to avoid spellchecker from seeing them?

Reply:
Done.